web/src/main/webapp/dataViews/tableView.ts

```diff
@@ -84,11 +85,17 @@ export class TableView extends TSViewBase implements IScrollTarget, OnNextK {
     protected message: HTMLElement;
     protected strFilter: StringFilterDescription;
 
+    protected privacySchema: PrivacySchema;
+
     // The following elements are used for Find
     protected findBar: FindBar;
 
     public constructor(
-        remoteObjectId: RemoteObjectId, rowCount: number, schema: SchemaClass, page: FullPage) {
+        remoteObjectId: RemoteObjectId,
```

Comment:
alignment

Reply:
What do you mean by this?

Reply:
If the diff I saw one of the arguments was indented with a different amount.